src/module.c

```diff
@@ -10141,6 +10159,9 @@ int moduleUnload(sds name) {
     } else if (module->blocked_clients) {
         errno = EAGAIN;
         return C_ERR;
+    } else if (moduleHoldsTimer(module)) {
+        errno = EINPROGRESS;
+        return REDISMODULE_ERR;
```

Comment:
```suggestion
        return C_ERR;
```

Reply:
I guess this is why your test fail.

Reply:
thanks I have updated to the correct one, but from my local test, the test was still failed.